### DISCUSSION TITLE
Virtual Router creating only on Local Storage

### DISCUSSION THREAD

**Discussion by happyalexkg** [apache/cloudstack#10107] [Q&A]
I created a Service Offering - System Offering For Software Router - HA with the following settings:

**Storage tags: nvme
Host tags: cloud**
Then, I copied the UUID of System Offering For Software Router - HA and pasted it into:

UUID of the service offering used by virtual routers; if NULL, the system offering will be used.
I restarted cloudstack-management.

However, when creating a new network, a new virtual router is created, and it still shows:

Compute offering: System Offering For Software Router - Local Storage.
The flag (System vm use local storage (system.vm.use.local.storage)) is disabled.

How can this be fixed?

**Comment by DaanHoogland** [comment]
@happyalexkg , How are the storage and host tags relevant?

the rest of the settings you describe seems ok, so some information must be missing.

**Comment by happyalexkg** [comment]
Hi, 
host tags:
![host_tag_server1](https://github.com/user-attachments/assets/badfad65-61e0-452e-8e9c-63355ccc72ef)
![host_tag_server2](https://github.com/user-attachments/assets/3315feb3-f73d-4646-9403-5d81ad8ef33f)
storage tag
![storage](https://github.com/user-attachments/assets/29883536-6c7d-4d33-a339-0982e5d6d4fb)

**Comment by DaanHoogland** [comment]
@happyalexkg , i have been blatently forgetting about your query. Is it still relevant?

If so is the offering being used the default systemvm one?

**Comment by DaanHoogland** [comment]
@happyalexkg , I am closing this thread. Please feel free to re-open if you have more information to add.
